nebu/cli/mathify.py

```diff
@@ -0,0 +1,100 @@
+import os
+from pathlib import Path
+import re
+
+import click
+import docker
+import requests
+
+from ..logger import logger
+from ._common import common_params
+
+
+IMAGE_TAG = 'openstax/cnx-bakedpdf'
+REPO_URL = 'https://github.com/openstax/baked-pdf.git'
+INPUT_FILE = 'collection.xhtml'
+OUTPUT_FILE = 'collection.mathified.xhtml'
+MOUNT_POINT = '/out'
+
+
+def get_input_file_path(collection_path):
+    for root, dirs, files in os.walk(collection_path):
+        if INPUT_FILE in files:
+            return Path(root, INPUT_FILE)
+
+
+@click.command()
+@common_params
+@click.option('-f', '--format', default='svg',
+              help='The format that math should transform to')
+@click.option('--build', is_flag=True,
+              help='Force build the baked-pdf image even if the image exists')
+@click.argument('collection_path')
+@click.pass_context
+def mathify(ctx, collection_path, build, format='svg'):
+    """Transform math in COLLECTION_PATH to svg or html"""
+    colpath = Path(collection_path)
+    input_file = get_input_file_path(collection_path)
+    if not input_file:
+        # invoke neb assemble if input file is not found
+        from .main import assemble  # TODO import from .assemble when ready
```

Comment:
If `collection.xhtml` doesn't exist, `neb mathify` calls `neb assemble`.  But the cli code for `neb assemble` doesn't exist yet so the tests are failing.